pkg/sqlite/load.go

```diff
@@ -921,36 +928,61 @@ func (s *sqlLoader) addBundleProperties(tx *sql.Tx, bundle *registry.Bundle) err
 		if err != nil {
 			return err
 		}
-		if err := s.addProperty(tx, registry.GVKType, string(value), bundle.Name, bundleVersion, bundle.BundleImage); err != nil {
-			return err
+		properties[propstring{Type: registry.GVKType, Value: string(value)}] = struct{}{}
+	}
+
+	// Add properties from annotations
+	csv, err := bundle.ClusterServiceVersion()
+	if err != nil {
+		// FIXME: Returning nil here is in line with the original implementation, but that was probably wrong. We should probably just bubble-up the error.
+		return nil
+	}
+
+	if csv == nil {
+		// FIXME: Currently, a CSV is requirement of bundle addition. Should this return an error?
+		return nil
```

Comment:
I agreed with the FIXME, CSV is required so if it is missing, an error is appropriate.